python/cudf/cudf/_lib/pylibcudf/copying.pyx

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+from cython.operator cimport dereference
+from libcpp.memory cimport unique_ptr
+from libcpp.utility cimport move
+
+# TODO: We want to make cpp a more full-featured package so that we can access
+# directly from that. It will make namespacing much cleaner in pylibcudf. What
+# we really want here would be
+# cimport libcudf... libcudf.copying.algo(...)
+from cudf._lib.cpp cimport copying as cpp_copying
+from cudf._lib.cpp.column.column_view cimport column_view
+from cudf._lib.cpp.table.table cimport table
+from cudf._lib.cpp.table.table_view cimport table_view
+
+from .column cimport Column
+from .table cimport Table
+
+
+# Cython doesn't support scoped enumerations. It assumes that enums correspond
+# to their underlying value types and will thus attempt operations that are
+# invalid. This code will ensure that these values are explicitly cast to the
+# underlying type before casting to the final type.
```

Comment:
Likely there's going to be tons of places we have to do this. Perhaps we document this pattern in the dev guide instead?

Reply:
Added a note about this in the pylibcudf dev docs PR (#13639) and removed all instances of this comment from cudf.